### DISCUSSION TITLE
Any similar Rigid element type like R3D4 in abaqus?

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#1492] [Contributors]
Hi everyone,

May I ask if there is any similar rigid element type in Radioss like R3D4 element in abaqus?

Thanks for help

Bests
Qinan

**Comment by danFrodo** [comment]
Hi Qinan, I moved from Abaqus-explicit to Radioss and this confused me also until I got some training.  To do what is typically done in Abq w R3D4 you need two objects:  A facet body that would be same thing you'd make in Abq of R3D4.  Then make an RBODY using those same nodes.  The facet body should have VOID material & property.  I have heard they are going to make R3D4 type Radioss feature but haven't seen it yet.

**Comment by danFrodo** [comment]
Here's a simple 'floor' for drop test that has single quad facet + RBODY.  I cut this out of a model I currently use.

[floor_0000.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/11559566/floor_0000.rad.txt)

**Reply by li2qn** [reply]
Many thanks for your detailed inputs Dan. This help out a lot. I am also doing drop test in Radioss. May I ask if you are able to share your complete files for my reference? I am trying to get my first job up and run. Thanks again.

**Comment by danFrodo** [comment]
Have you looked at the drop test in the examples from openradioss?

https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/11141156/Cell+Phone+Drop+Test
